App/static/js/content-recommendation.js

```diff
@@ -0,0 +1,379 @@
+function recommendBooks() {
+    let element = document.getElementById("force_graph");
+
+    if (element != null) {
+        element.remove();
+    }
+
+    let query = document.getElementById("user_query");
+
+    if (query.value === '') {
+        alert("Please enter some text before querying!")
+    }
+    else {
+        const url = `http://localhost:5000/recommend/${query.value}`
+        fetch(url)
+            .then(response => response.json())
+            .then(json => {
+                json['query'] = {
+                    'score': 0,
+                    'title': query.value,
+                    'pandas_index': -1,
+                }
+                generate_force_graph(json);
```

Comment:
@ZebraAlgebra you take the json response from your API route and then pass it to a normal JS function